packages/mui-joy/src/Breadcrumbs/Breadcrumbs.test.js

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createRenderer, describeConformance } from 'test/utils';
+import { ThemeProvider } from '@mui/joy/styles';
+import Breadcrumbs, { breadcrumbsClasses as classes } from '@mui/joy/Breadcrumbs';
+import { unstable_capitalize as capitalize } from '@mui/utils';
+
+describe('<Breadcrumbs />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Breadcrumbs />, () => ({
+    classes,
+    inheritComponent: 'nav',
+    render,
+    ThemeProvider,
+    muiName: 'JoyBreadcrumbs',
+    refInstanceof: window.HTMLElement,
+    testVariantProps: { variant: 'solid' },
+    skip: [
+      'classesRoot',
+      'componentProp',
+      'componentsProp',
+      'themeDefaultProps',
+      'themeVariants',
+      'themeStyleOverrides',
```

Comment:
These should not be skipped for the breadcrumbs, right?

Reply:
How do we know which can be skipped and which can't?

Reply:
Only skip the things that Joy does not support. As far as I remember, the `classesRoot` (`classes` prop).

Reply:
Got it. I changed it to `skip: ['classesRoot', 'componentsProp'],`.

I get 1 error now. Can you teach me how to fix this?

![Screenshot 2022-08-29 at 14 35 48](https://user-images.githubusercontent.com/32841130/187213808-473d21d0-8aef-4991-a17a-9de2ff17f819.png)

Reply:
It throws an error because the `muiName` does not match the test:

```js
// test
describeConformance({
  ...
  muiName: 'JoyBreadcrumbs',
  ...
})

// Breadcrumbs.tsx
const BreadcrumbsRoot = styled('nav', {
  name: 'MuiBreadcrumbs', // <<<<<<
  slot: 'Root',
  overridesResolver: (props, styles) => styles.root,
})
```

Reply:
Thank you Jun!!